static/js/vue-cdr-access/src/components/full_record/breadCrumbs.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+    <div id="full_record_trail">
+        <span class="hierarchicalTrail">
+            <template v-for="(path, index) in objectPath.entries">
+                <template v-if="index === (breadcrumbSize - 1)">
+                    <span class="quote">&raquo;</span>
+                    <span>{{ path.name }}</span>
+                </template>
+                <template v-else-if="index <= 2 || breadcrumbSize <= 5 || (index === breadcrumbSize - 2)">
+                    <a v-if="index === 0" href="/collections">{{ $t('breadcrumbs.collections') }}</a>
+                    <template v-else>
+                        <span class="quote">&raquo;</span>
+                        <a :href="shiftFacetUrl(path.pid)">{{ path.name }}</a>
+                    </template>
+                </template>
+                <template v-else>
+                    <template v-if="index === 3 && !showFullBreadCrumb">
+                        <span class="quote">&raquo;</span>
+                        <a id="expand-breadcrumb" @click.prevent="showFullCrumb" href="#">&hellip;</a>
+                    </template>
+                    <template v-else>
+                        <span class="full-crumb quote">&raquo;</span>
+                        <a :href="shiftFacetUrl(path.pid)" class="full-crumb">{{ path.name }}</a>
+                    </template>
+                </template>
+            </template>
+        </span>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'breadCrumbs',
+
+    props: {
+        objectPath: Object,
+        queryPath: {
+            type: String,
+            default: 'record'
+        }
+    },
+
+    data() {
+        return {
+            showFullBreadCrumb: false
+        }
+    },
+
+    computed: {
+        breadcrumbSize() {
+            return this.objectPath.entries.length;
+        },
+
+        shiftFacetUrlBase() {
+            if (this.searchStateUrl === '') {
+                return '';
+            }
+            return `/${this.searchStateUrl}`;
+        },
+
+        searchStateUrl() {
+            return window.location.search;
+        }
+    },
+
+    methods: {
+        shiftFacetUrl(pid) {
+            return `${this.queryPath}/${pid}${this.shiftFacetUrlBase}`;
```

Comment:
Looking at the behavior on prod, I don't think we're actually appending any of the searchStateUrl stuff to the end of the breadcrumb urls. When I hover over the crumb links in prod they stop at the pid. So I suspect we can remove the `shiftFacetUrlBase` method